en/create-staff.md

```diff
@@ -0,0 +1,85 @@
+
+![Alt text](../img/Screenshot%202022-08-30%20at%2013.29.30.png?raw=true "Sakurabook Add Staff")
+
+📌 All the required fields are labeled with `*`.
+
+## Basic information
+
+Fill in all the basic details about your staff.
+
+![Alt text](../img/Screenshot%202022-08-30%20at%2013.27.19.png?raw=true "Sakurabook Staff Basic information")
+
+| Field               | Required | Description                                                          |
+| ------------------- | -------- | -------------------------------------------------------------------- |
+| `First name`        | x        | First name of the staff.                                             |
+| `Last name`         | x        | Last name of the staff.                                              |
+| `Email`             | x        | Email address in a valid format.                                     |
+| `Phone`             | x        | Phone number in a valid format, including country code prefix.       |
+| `Bio`               |          | Details about the staff.                                             |
+| `Assigned services` |          | Click on the `Add service` button to connect staff to your services. |
+| `Photo`             |          | Profile picture of the staff.                                        |
+|                     |          |                                                                      |
+
+## Business hours
+
+Fill in the hours when your staff is available. These are the times that are offered to your customers. Bookings cannot be created outside of business hours.
+
+Use the `Nonstop` if the staff is available all the time without any breaks. Use `Custom` to create time windows. Use the bin button (🗑) to delete a specific slot. Use the plus button (➕) to add another time slot within the specific day. Use the copy button to copy a specific day time slot into another day/s.
+
+![Alt text](../img/Screenshot%202022-08-30%20at%2013.27.47.png?raw=true "Sakurabook Staff Business hours")
+
+## Prepare
+
+Do your staff need some time before/after an appointment? Fill in this section. If not, feel free to fill in `0`.
+
+![Alt text](../img/Screenshot%202022-08-30%20at%2013.28.01.png?raw=true "Sakurabook Staff Prepare")
+
+| Field             | Required | Description                                                                   |
+| ----------------- | -------- | ----------------------------------------------------------------------------- |
+| `Before check-in` | x        | How much time your staff needs before each appointment, e.g. for preparation? |
+| `After check-out` | x        | How much time your staff needs after the appointment, e.g. to rest?           |
+
+## Staff location
+
+Fill in as many locations as you want. These locations are meant to be the places where your staff is available. Use location autocomplete to search for correct locations. Use the `In person` checkbox if your staff is available for in-person appointments.
```

Comment:
I would add the info, that customers will be able to choose any of these locations.